daemons/execd/remoted_pidone.c

```diff
@@ -156,11 +157,17 @@ load_env_vars(void)
          */
         value_end = end;
 
+        // Strip trailing comment beginning with '#'
+        comment = strchr(end, '#');
```

Comment:
Technically this is wrong in the case of, say, `name=value# asdf`. It turns out Bash assigns sets `name` to `"value#"`, not `"value"`. So the `asdf` would be treated as a second token, not as part of a comment, and the whole line would be invalid.

Using `g_shell_parse_argv()` in a later commit fixes that among other things.